ProjectSpace/explore/src/main/java/com/pwc/explore/RealTimeTest.java

```diff
@@ -27,13 +28,20 @@ public RealTimeTest() {
     public float getAccuracy() { return accuracy; }
     public void setAccuracy() { this.accuracy = (float) (this.numOfCorrect * 100) / this.numOfTry; }
 
-    public void saveResult(Context context) {
+    public void saveResult(Context context, Boolean isInternal) {
         String TAG = "RealTimeTest";
         try {
-            FileWriter out = new FileWriter(new File(context.getFilesDir(), "testResult.txt"));
+            if (isInternal) {
+                FileWriter out = new FileWriter(new File(context.getFilesDir(), "testResult.txt"));
+                out.write(toString());
+                out.close();
+            }
+            else {
+                FileWriter out = new FileWriter(new File(context.getExternalFilesDir(Environment.DIRECTORY_DOCUMENTS), "testResult.txt"));
+                out.write(toString());
+                out.close();
+            }
```

Comment:
By @Kalaiz's suggestion, I added the option for external storage.

Reply:
Looks good!